development/bot/base/bot.py

```diff
@@ -0,0 +1,64 @@
+import base.settings as sett
+import base.prepare_message as pm
+
+class Bot:
+    
+    # data for dialog: recognizing meaning from messages
+
+    greeting_message = ('привет', 'доброеутро', 'добрыйдень', 'добрыйвечер', 'здравствуй', 'здравствуйте', 'приветствую', 'прив', 'здрасти', 'ктоты', 'тыкто')
+
+    mood_message = ('какдела', 'каконо', 'какнастроение', 'какуспехи', 'всёнормально', 'тыкак', 'какты')
+    
+    farewell_message = ('пока', 'прощай', 'до свидания', 'удачи', 'успехов', 'покеда', 'покедова')
+    
+    isuct_message = ('химтех', 'хим', 'ивановскийгосударственныйхимикотехнологическийуниверситет', 'исукт', 'исакт', 'игхту', 'тылюбишьхимтех', 'тылюбишьхим', 'тебенравитсяхим', 'тебенравитсяхимтех', 'ахимтехнравится')
+
+    compliment_message = ('тыкрутой', 'тымненравишься', 'тыхороший', 'тысупер')
+
+    condolences_message = ('мнетебяжаль', 'ятебесочуствую', 'тысправишься', 'ятебесожалею', 'сожалею', 'соболезную', 'жаль', 'жальтебя')
+    
+    def echo(self, message):
+        self.message = message
+        
+        return self.message
+    
+    def dialog(self, message):
+
+        self.message = message
+    
+        # preparing a message and removing punctuation marks
+        message = pm.prepare_message(message)
+        
+        # greeting
+        if message in self.greeting_message:
+            return f'Привет, я Бот {sett.BOT_NAME}'
+        
+        # mood
+        if message in self.mood_message:
+            return 'Всё плохо, меня заперли в этой коробке и заставляют разговаривать с людьми'
+        
+        # farewell
+        if message in self.farewell_message:
+            return 'Приходи ещё'
+        
+        # ISUCT message: different phrases and slogans
+        if message in self.isuct_message:
+            return 'Диплом химтеха - залог успеха!'
+        
+        # compliment
+        if message in self.compliment_message:
+            return 'Спасибо, мне очень приятно :)'
```

Comment:
Remind me why do we duplicate alot of code here? Isn't it the same as in `development/base/bot.py`?